test/e2e/clustergroup_test.go

```diff
@@ -52,7 +52,7 @@ func testInvalidCGIPBlockWithPodSelector(t *testing.T) {
 func testInvalidCGIPBlockWithNSSelector(t *testing.T) {
 	invalidErr := fmt.Errorf("clustergroup created with ipblock and namespaceSelector")
 	cgName := "ipb-ns"
-	nSel := &metav1.LabelSelector{MatchLabels: map[string]string{"ns": "y"}}
+	nSel := &metav1.LabelSelector{MatchLabels: map[string]string{"ns": "ycg"}}
```

Comment:
This is too obscure to understand how "ycg" is generated, could we get the namespace name from some variables?